pre-aio-node-with-zuul-ref.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+set -ex
+refspec="$1"
+ZUUL_CHANGES="$2"
+keep_env="$3"
+
+FETCH_URL=http://10.71.212.50/p/
+ROOT_DIR=/opt/stack/
+DEVSTACK_TOOL=/opt/stack/devstack
+NOVA_DIR=/opt/stack/nova
+TEMPEST_DIR=/opt/stack/tempest
+TMP_MERGE_BRANCH=tmp_branch_for_vgpu_patches
+DEST=${DEST:-"/opt/stack/openstack"}
+NOVA_CONF_PATH=/etc/nova
+SAMPLE_IMAGE=cirros-0.3.5-x86_64-disk
```

Comment:
=>IMAGE_NAME=${IMAGE_NAME:-cirros-0.3.5-x86_64-disk}

Reply:
Will change it